File: backend_healthconnect/src/main/java/com/backend_healthconnect/model/utenteDTO.java
package com.backend_healthconnect.model;

import lombok.*;

import java.time.LocalDate;
import java.time.LocalDateTime;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class utenteDTO {
    private Long id;
    private String nome;
    private String cognome;
    private String email;
    private String password;
    private Long telefono;
    private LocalDate dataNascita;
    private Ruolo ruolo;
    private LocalDate dataCreazione;
    private String sesso;

    //attributi dettaglio medico
    private Long specializzazione_id;
    private String numero_albo;
    private String biografia;
    private String indirizzo_studio;
    private StatoApprovazione stato_approvazione;
}

File: frontend_healthconnect/src/app/model/utenteDTO.ts

export interface utenteDTO{
  id:number
  email: string
  nome: string
  cognome: string
  telefono: number
  dataNascita: string
  ruolo: string
  dataCreazione: string
  sesso: string

  specializzazione_id?: number;
  numero_albo?: string;
  biografia?: string;
  indirizzo_studio?: string;
  stato_approvazione?: 'PENDING' | 'APPROVATO' | 'RIFIUTATO';
}
